client/src/components/About.js

```diff
@@ -1,17 +1,17 @@
 import React from "react";
-
+import { Container, Header } from "semantic-ui-react";
 const About = () => {
   return (
-    <div>
-      About
+    <Container>
+      <Header as="h2">About</Header>
```

Comment:
It is better to use h1 instead of h2 because it is the first title on the page.

Reply:
done